src/dotnet-compile-fsc/project.json

```diff
@@ -3,9 +3,14 @@
   "compilationOptions": {
     "emitEntryPoint": true
   },
+  "compile": [
+    "**/*.cs",
+    "../dotnet/CommandLine/*.cs",
+    "../dotnet/commands/dotnet-compile-csc/AssemblyInfoOptionsCommandLine.cs",
```

Comment:
This command is going to move out to another repo in the near future, can we share these types in Microsoft.Compiler.Common?